docs/howto/configure/advanced/extensions.md

```diff
@@ -82,7 +79,7 @@ If detected, [`libarchive-c`](https://pypi.org/project/libarchive-c) will be use
 better performance, especially when working with archives with many/large assets,
 espcially [pyodide](../../python/pyodide.md).
 
-If not `libarchive-c` is not detected, python's built-in `zipfile` and `tarfile` modules
+If `libarchive-c` is not detected, Python's built-in `zipfile` and `tarfile` modules
```

Comment:
we should move away from `libarchive-c`: outside of `conda/mamba`, it is a pain to install... i don't know what to replace it with, though... the _whole idea_ is to be able to use binary tools for vetted performance and security, but every one of them carries complexity risks.

Reply:
Let's track that in an issue?